src/TabSanity.ts

```diff
@@ -230,39 +230,4 @@ export class TabSanity {
 		}, this));
 	}
 
-	public async deleteLeft() {
-		for (let selection of this.editor.selections) {
-			const start = selection.start;
-			if (!selection.isEmpty) {
-				await this.delete(selection);
-				continue;
-			}
-			const deleteStartPosition = this.findNextLeftPosition(start);
-			if (deleteStartPosition) {
-				await this.delete(new Range(deleteStartPosition, start));
-			}
-		}
-		this.editor.revealRange(this.editor.selection);
-	}
-
-	private delete(location: Range|Selection) {
-		return this.editor.edit(edit => {
-			edit.delete(location);
-		});
-	}
-
-	public async deleteRight() {
```

Comment:
@PeterCP can we discuss this change? I see that vscode now deletes left out-of-the-box, but it appears **_not_** to delete right.

Might have to restore this function.

Reply:
I didn't pay that much attention to the delete right :s sorry! That was my bad.

Reply:
All good. Just making sure I didn't miss something.

Reply:
Oh and I restored delete right, so we're good.